jpa/hibernate-transformer/src/main/java/org/jboss/as/hibernate/TransformerLogger.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2018 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.as.hibernate;
+
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.Logger;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * TransformerLogger
+ *
+ * @deprecated
+ */
+@MessageLogger(projectCode = "ORMTRFMR")
+public interface TransformerLogger extends BasicLogger {
+    TransformerLogger LOGGER = Logger.getMessageLogger(TransformerLogger.class, "org.jboss.as.hibernate.transformer");
```

Comment:
I think since we're only using debug/trace messages we likely don't need a logging interface.

Reply:
good point, thanks @jamezp

Reply:
@jamezp which logger should we use instead?  I naively tried:
```
public class TransformerLogger extends DelegatingBasicLogger implements BasicLogger {
    public static final TransformerLogger LOGGER = Logger.getMessageLogger(TransformerLogger.class, "org.jboss.as.hibernate.transformer");

    public TransformerLogger(final Logger log) {
        super(log);
    }
}

Reply:
Looks like Logger.getLogger(String name) should work out...